### PR TITLE
[type:fix] namespace user rel

### PR DESCRIPTION
**Pull request by Aias00**
namespace bugs fix

Make sure that:

- [x] You have read the [contribution guidelines](https://shenyu.apache.org/community/contributor-guide).
- [x] You submit test cases (unit or integration tests) that back your changes.
- [x] Your local test passed `./mvnw clean install -Dmaven.javadoc.skip=true`.

### CODE REVIEW

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/model/event/namespace/NamespaceCreatedEvent.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+     * @param userId operator
+     */
+    public NamespaceCreatedEvent(final NamespaceDO source, final String userId) {
+        super(source, null, EventTypeEnum.NAMESPACE_CREATE, null, userId);
+    }
+    
+    
+    @Override
+    public String eventName() {
+        return "namespace";
```
Comment (moremind) [inline shenyu-admin/src/main/java/org/apache/shenyu/admin/model/event/namespace/NamespaceCreatedEvent.java:42]: not use magic value

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/model/event/namespace/NamespaceCreatedEvent.java (hunk 1 above)
Comment (Aias00) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/model/event/namespace/NamespaceCreatedEvent.java:42]: but all event class like this: 
image

Comment (moremind) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/model/event/namespace/NamespaceCreatedEvent.java:42]: you can refactor the magic value

File: db/init/oracle/schema.sql [hunk 2]
```diff
@@ -2997,4 +2997,27 @@ comment on column SCALE_HISTORY.msg
...
+
+create table namespace_user_rel
+(
+    id             varchar(128) not null,
+    namespace_id   varchar(50) NOT NULL,
+    user_id        varchar(128) NOT NULL,
+    date_created   timestamp(3) default SYSDATE not null,
+    date_updated   timestamp(3) default SYSDATE not null,
+    PRIMARY KEY (id)
+);
```
Comment (moremind) [inline db/init/oracle/schema.sql:3011]: you shoud add the script to upgrade script and you should add this to h2 sql

**Review by moremind — CHANGES_REQUESTED** [review]

File: shenyu-admin/src/main/java/org/apache/shenyu/admin/model/event/namespace/NamespaceCreatedEvent.java (hunk 1 above)
Comment (Aias00) [reply shenyu-admin/src/main/java/org/apache/shenyu/admin/model/event/namespace/NamespaceCreatedEvent.java:42]: ok, i will do that

File: db/init/oracle/schema.sql (hunk 2 above)
Comment (Aias00) [reply db/init/oracle/schema.sql:3011]: oh, i missed

**Review by moremind — APPROVED** [review]

**Pull request merged**